.github/workflows/zowe-cli-plugin.yml

```diff
@@ -12,7 +12,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        node-version: [14.x, 16.x, 18.x]

```

Comment:
now that node 14 is gone. we could remove the `npm@8` update step from the workflow since node16 comes with npm v8